en/mdn_plus_privacy.md

```diff
@@ -31,10 +31,9 @@ MDN Plus doesn’t share any of your data with third parties.
 
 ### Data privacy questions? {: #markup }
 
-Much of the information that we store about our Firefox Account users is easily accessible by logging into your account. You can learn more about  managing your data [here](https://support.mozilla.org/products/privacy-and-security/user-control). To make requests, please contact us through our [Data Subject Access Request Portal](https://privacyportal.onetrust.com/webform/1350748f-7139-405c-8188-22740b3b5587/4ba08202-2ede-4934-a89e-f0b0870f95f0).
+Much of the information that we store about our Firefox Mozilla Account users is easily accessible by signing in to your account, where you can also update your data sharing settings. To make requests regarding your personal data, please contact us through our [Data Subject Access Request Portal](https://privacyportal.onetrust.com/webform/1350748f-7139-405c-8188-22740b3b5587/4ba08202-2ede-4934-a89e-f0b0870f95f0).
 
 If you have any other questions regarding our privacy practices, please contact us at compliance@mozilla.com.
-
-We respond to all requests we receive from individuals wishing to exercise their data protection rights in accordance with applicable data protection laws.
+We respond to all requests we receive from individuals wishing to exercise their data protection rights regardless of where the individual lives. We will honor your request unless a legal requirement prevents us from doing so or a legal exception applies.
```

Comment:
If this is a separate paragraph, add a line space. Otherwise, wrap it to where the email address ends.

Reply:
Done